recOrder/tests/util_tests/test_create_empty.py

```diff
@@ -31,6 +31,18 @@ def test_create_empty_hcs_zarr():
             assert "Channel2" in z.channel_names
 
     # Repeat creation should not fail
+    more_channel_names = ["Channel3"]
     create_empty_hcs_zarr(
-        store_path, position_keys, shape, chunks, scale, channel_names, dtype
+        store_path,
+        position_keys,
+        shape,
+        chunks,
+        scale,
+        more_channel_names,
+        dtype,
     )
+
+    # Verify existence of appended channel names
+    with open_ome_zarr(store_path, mode="r") as z:
+        for position_key in position_keys:
+            assert "Channel3" in z.channel_names
```

Comment:
The old test (L30-31) was not testing each position, but checking the same plate metadata P times.

Reply:
@talonchandler note that I changed C shape in https://github.com/mehta-lab/recOrder/pull/454/commits/ecf1b8a806798ae7fec69ecbbac2e544386c9723.

Reply:
Great catch!